src/VisualStudio/Core/Def/Packaging/Interop/SVsRemoteControlService.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Internal.VisualStudio.Shell.Interop
+{
+    [Guid("BF443850-E346-44A8-B03C-11B15ACDEEC1")]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    internal interface SVsRemoteControlService
```

Comment:
Why can't we reference 14.1? Did it depend on getting Update 1 on the build machines (which is now done), or something else?